common/reviews/api/core.api.md

```diff
@@ -2751,6 +2788,9 @@ export class VolumeViewport3D extends BaseVolumeViewport {
 type VolumeViewportProperties = {

```

Comment:
Can you define a common class for shared properties between stack and volume?  Things like invert and colormap probably should be defined once rather than multiple times.